### PR TITLE
Blog App: validations and model specs

### PR DESCRIPTION
**Pull request by enis-memic**
### Add the following validations:

-  For the User model:
      Name must not be blank.
      PostsCounter must be an integer greater than or 
      equal to zero.

-  For the Post model:
     Title must not be blank.
     Title must not exceed 250 characters.
     CommentsCounter must be an integer greater 
     than or equal to zero.
     LikesCounter must be an integer greater than or 
     equal to zero.

- Add unit specs for all of your models' methods and validations.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -3,6 +3,9 @@ class User < ApplicationRecord
   has_many :comments, foreign_key: 'author_id'
   has_many :posts, foreign_key: 'author_id'
 
+  validates :name, presence: true
+  validates :post_counter, numericality: { only_integer: true }, comparison: { greater_than_or_equal_to: 0 }
+
   def recent_posts
     posts.order(created_at: :desc).limit(3)
   end
```
Comment (OyePriscilla) [inline app/models/user.rb:11]: - [x] Please kindly implement unit test for this method as well according to the project requirement. 👍

File: app/models/post.rb [hunk 2]
```diff
@@ -3,12 +3,16 @@ class Post < ApplicationRecord
...
   def recent_comments
     comments.order(created_at: :desc).limit(5)
   end
 
   def update_posts_counter
-    author.increment!(:posts_counter)
+    author.increment!(:post_counter)
   end
   after_save :update_posts_counter
 end
```
Comment (OyePriscilla) [inline app/models/post.rb:18]: - [x] Please kindly implement unit test for all the methods in post model as well according to the project requirement. 👍

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @enis-memic 👍, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Implemented validations for models ✔️ 
Write unit tests for some of the models' methods ✔️ 

## Required Changes ♻️

- [x] @enis-memic **Please kindly fix your license link (README file); I got 404 error at the moment. 👍  Fixing it will make your README looks professional.** ℹ️ 

- [x] Kindly ensure to implement unit test for all the methods in post and user model accordingly. **You did well implementing unit test for the methods in comment and likes models.** 👍 👍 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @enis-memic 

Your project is complete! Great job for implementing the previous reviewers changes 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- Validations correctly implemented
- Test well added
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

kd

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**